js/interface.js

```diff
@@ -60,10 +60,25 @@ Fliplet.Widget.register('com.fliplet.theme', function() {
     }
 
     if (!themesLoadingPromise) {
-      themesLoadingPromise = Fliplet.Themes.get().then(function (response) {
+      themesLoadingPromise = Fliplet.API.request({
+        url: [
+          'v1/widgets?include_instances=true&tags=type:theme',
+          '&include_all_versions=true',
+          '&appId=' + F.Env.get('appId'),
```

Comment:
`F` is undefined here, it should be `Fliplet`. Did you test this? :)

Reply:
@squallstar 🙄 
.
.
.
No I didn't! 😁